cmd/sonobuoy/app/run.go

```diff
@@ -71,6 +75,15 @@ func NewCmdRun() *cobra.Command {
 }
 
 func submitSonobuoyRun(cmd *cobra.Command, args []string) {
+
+	var s *spinner.Spinner
+
+	if (runflags.wait > 0 && runflags.waitOutput == "spinner") {
+		s = getSpinnerInstance()
+		s.Start()
```

Comment:
I'm on the fence about the desire to just put this snippet in a goroutine and sleep for 2-3s just to avoid the first line clash between the spinner and the log output.

Maybe we just leave it this way for now (again it defaults to silent) and see how it goes. We can always do that followup, especially if we ever want to make it the default.